packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,203 @@
+    height: var(--terra-slider-track-height, 0.43rem);
+  }
+
+  .input-range::-ms-thumb {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: var(--terra-slider-thumb-background, #007cc3);
+    border-color: var(--terra-slider-thumb-border-color, #0065a3);
+    border-radius: var(--terra-slider-thumb-border-radius, 1.715rem);
+    border-style: solid;
+    border-width: var(--terra-slider-thumb-border-width, 0.071rem);
+    box-shadow: var(--terra-slider-thumb-box-shadow, 0 0 0.286rem 0 rgba(0, 0, 0, 0.25));
+    cursor: pointer;
+    height: var(--terra-slider-thumb-height, 1.715rem);
+    margin-top: -0.57rem;
+    width: var(--terra-slider-thumb-width, 1.715rem);
+  }
+
+  .input-range:focus::-ms-thumb {
+    outline: var(--terra-slider-thumb-focus-outline, 0.143rem dashed #000);
+    outline-offset: 0.071rem;
+  }
+
+  .input-range:hover::-ms-thumb {
+    background: var(--terra-slider-thumb-hover-background, #0065a3);
+  }
+
+  .input-range:active::-ms-thumb {
+    background: var(--terra-slider-thumb-active-background, #004c75);
+  }
+
+  .input-range:hover::-ms-track {
+    background: linear-gradient(to right, var(--terra-slider-track-selected-range-hover, #0065a3) 0%, var(--terra-slider-track-selected-range-hover, #0065a3) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range-hover, #bcbfc0) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range-hover, #bcbfc0) 100%);
+  }
+
+  .input-range:disabled::-ms-track {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: linear-gradient(to right, var(--terra-slider-track-disabled-selected-range, #007cc3) 0%, var(--terra-slider-track-disabled-selected-range, #007cc3)  50%, var(--terra-slider-track-disabled-unselected-range, #d3d4d5)  50%, var(--terra-slider-track-disabled-unselected-range, #d3d4d5) 100%);
+  }
+
+  .input-range:disabled::-ms-thumb {
+    -webkit-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: var(--terra-slider-thumb-disabled-background, #007cc3);
+    cursor: default;
+  }
+
+  /* For Firefox */
+  .input-range::-moz-range-track {
+    -moz-appearance: none; /* stylelint-disable-line property-no-vendor-prefix, plugin/no-unsupported-browser-features */
+    background: linear-gradient(to right, var(--terra-slider-track-selected-range, #007cc3) 0%, var(--terra-slider-track-selected-range, #007cc3) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) var(--terra-slider-progress-status, 50%), var(--terra-slider-track-unselected-range, #d3d4d5) 100%);
+    border-radius: var(--terra-slider-track-border-radius, 0.214rem);
+    height: var(--terra-slider-track-height, 0.43rem);
+  }
+
+  .input-range::-moz-range-thumb {
```

Comment:
Aren't you able to combine some of these selectors and their corresponding styles so that there is not so much duplication?

Reply:
Logged UXPLATFORM-9778 for future css optimizations.